cli/internal/helm/charts/edgeless/constellation-services/charts/kms/values.yaml

```diff
@@ -1,13 +1,13 @@
 # Namespace to which KMS will be deployed.
 namespace: "kube-system"
 # Port on which the service will listen.
-kmsPort: 9000
+port: 9000
 # Name of the ConfigMap that holds measurements and other info.
 joinConfigCMName: join-config
 # Path to which secrets/CMs are mounted.
 serviceBasePath: /var/config
 # Container image.
-kmsImage: setFullImagePathHere
+image: setFullImagePathHere
```

Comment:
I guess this is a question for everyone:
Do we want to set our images here? Or would it be preferable to pass the values for images from Go code?

Reply:
Just for completeness: they are being passed from Go code atm.

https://github.com/edgelesssys/constellation/pull/341/files#diff-540be4cd28ea6a60b8ecfc4c864805bdcb896893cb33f121bfad873a878e2783R110

Reply:
I am kinda torn on this one: bit I think I lean a bit more to moving them from `versions.go` into the helm charts to be able to use the charts standalone.

Reply:
I like the versions.go file as it makes finding version information very easy.

Reply:
Thoughts on just setting the image to `latest` in the helm charts, but keeping the actual used values in `versions.go`?
Since the most common use case for standalone helm charts is probably managing updates, users would need to adjust the image version in any case

Reply:
Thinking about this a bit more I like the fact that this doesn't fail silently. You are forced to think about the image version you want to deploy. Isn't that useful to have?

Reply:
Is there such a thing as a "required" field for helm values :thinking:

Reply:
Possible, but would remove the value from the values.yaml. Is that fine for you?

Reply:
Reading [the helm docs](https://helm.sh/docs/howto/charts_tips_and_tricks/#know-your-template-functions), it sounds to me as if we only need to have the value set to empty, and if not we throw an error.

This would result in even quicker errors (on deploy instead of image pull errors after deployment has already taken place)

Reply:
Using schemas is probably the most fleshed out solution here. Added just now.

Regarding values in the global scope: With the upcoming join-service migration it should become more apparent why I think it make sense to keep chart specific values in the respective chart dir. The global values file can then be used for shared values. [Branch](https://github.com/edgelesssys/constellation/commit/9caa23acbcc731789f2634ea085b549fe1ec7e4f#diff-e6021db951db1bd26edea8fcab79b3eda4edc89fa94ab5d4cd246e15566c7311).

Not super happy with the way the helm-lint target has to be implemented, but I guess it's manageable.